vm_config.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions for configuration of python environment."""
+
+import os
+import sys
+
+# Root path of the repo.
+ROOT_PATH = os.path.dirname(__file__)
```

Comment:
Oh, I get it now. It was working without `abspath` so I did not check what it contains. Thanks